ush/python/pygfs/task/jedi.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/env python3
+
+import os
+from logging import getLogger
+from pprint import pformat
+from typing import List, Dict, Any, Optional
+from jcb import render
+from wxflow import (AttrDict,
+                    chdir, rm_p,
+                    parse_j2yaml, save_as_yaml,
+                    logit,
+                    Task,
+                    Executable,
+                    WorkflowException)
+
+logger = getLogger(__name__.split('.')[-1])
+
+
+class JEDI:
+
+    def __init__(self, task_config: AttrDict[str, Any]) -> None:
+
+        _exe_name = os.path.basename(task_config.JEDIEXE)
+
+        self.exe = os.path.join(task_config.DATA, _exe_name)
+        self.yaml = os.path.join(task_config.DATA, os.path.splitext(_exe_name)[0] + '.yaml')
+        self.config = {}
```

Comment:
Perhaps want to initialize this as an `AttrDict` type.
```suggestion
        self.config = AttrDict()
```